packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Baker/Intro/BakerIntro.tsx

```diff
@@ -10,7 +10,7 @@ export default function BakerIntro() {
     const { t } = useTranslation('x', { keyPrefix: 'earn.validator.intro' });
     return (
         <Page className="baker-intro-container">
-            <Carousel onDone={() => nav(absoluteRoutes.settings.earn.baker.register.path)}>
+            <Carousel onDone={() => nav(absoluteRoutes.settings.earn.validator.register.path)}>
                 <span className="capture__main_small">
```

Comment:
I was not managed to update Delegator part, before you started working on it.
I want to ask you to replace such places like:
`<span className="capture__main_small"> with <Text.Capture>`
`<span className="text__main"> with <Text.Main>`
`<span className="heading_large"> with <Text.HeadingLarge>`

Later, I will go through the project again in search of similar remnants